packages/solid/tailwind.config.cjs

```diff
@@ -4,6 +4,9 @@ const colors = require("tailwindcss/colors");
 module.exports = {
   content: ["./src/**/*.{js,ts,jsx,tsx,mdx}"],
   prefix: "ut-",
+  corePlugins: {
+    preflight: false,
+  },
```

Comment:
Whats the difference between doing this here rather than not including the `@tailwind base` in the css file?

Reply:
Did some digging, looks like `@tailwindcss/forms` is also doing some global stuff that ill need to rework. Looking at tailwind's documentation, setting `preflight: false` seems to be the preferred way to turn off basic style resets.

Reply:
Did some further research, `@tailwindcss/forms` isnt actually styling anything in our library, since every `input` is hidden and of `type='file'` which forms doesnt touch regardless. Removed it as a dep.